garrysmod/gamemodes/terrortown/gamemode/shared.lua

```diff
@@ -143,7 +143,7 @@ local ttt_playercolors = {
 
 CreateConVar("ttt_playercolor_mode", "1")
 function GM:TTTPlayerColor(model)
-   local mode = GetConVarNumber("ttt_playercolor_mode") or 0
+   local mode = GetConVar("ttt_playercolor_mode"):GetInt() or 0

```

Comment:
Meh, I'd use the `ConVar` object returned by the `CreateConVar` literally two lines up but I guess this doesn't matter anymore now `GetConVar` caches things.